.pre-commit-config.yaml

```diff
@@ -70,19 +70,6 @@ repos:
     -   id: fix-encoding-pragma
         args: [--remove]
     -   id: trailing-whitespace
--   repo: https://github.com/cpplint/cpplint
```

Comment:
nanoarrow and cpplint have a ton of conflicts. generally not sure how useful cpplint is for us so just removed check